src/main/java/seedu/internbuddy/storage/JsonAdaptedCompany.java

```diff
@@ -15,54 +15,58 @@
 import seedu.internbuddy.model.company.Email;
 import seedu.internbuddy.model.company.Name;
 import seedu.internbuddy.model.company.Phone;
+import seedu.internbuddy.model.company.Status;
 import seedu.internbuddy.model.tag.Tag;
 
 /**
  * Jackson-friendly version of {@link Company}.
  */
-class JsonAdaptedCompany {
-
-    public static final String MISSING_FIELD_MESSAGE_FORMAT = "company's %s field is missing!";
+public class JsonAdaptedCompany {
```

Comment:
Curious but is there a need to be public instead of package private?

Reply:
This is the same as in the old `JsonAdaptedPerson` I think